explorer/client/src/theme.ts

```diff
@@ -43,7 +44,7 @@ const theme = {
       primary: '#3d5170',
       secondary: '#5a6169',
     },
-  },
+  } as PaletteOptions,
```

Comment:
Don't think we need this, we can instead type `const theme: ThemeOptions`. See `styleguide/src/theme.ts`

Reply:
https://github.com/smartcontractkit/chainlink/pull/2119#discussion_r363540675